Secrets/Models/Config.cs

```diff
@@ -30,6 +33,13 @@ class Config : ConfigFile
 
         [Toggle("Save Redemption Messages")]
         public bool SaveRedemptionMessages { get; set; } = false;
+
+        [Button("Auth With Twitch (Requires Restart)")]
+        public void MyCheckboxToggleEvent(ToggleChangedEventArgs e)
```

Comment:
Since this is a button now, we don't need the EventArgs and the name could be AuthWithTwitch or something like that...